File: sample-data-repository/src/main/java/nl/haploid/octowight/sample/data/CaptainModel.java
package nl.haploid.octowight.sample.data;

import nl.haploid.octowight.registry.data.Model;

public class CaptainModel implements Model {

	private Long id;

	private String name;

	public Long getId() {
		return id;
	}

	public String getName() {
		return name;
	}

	public void setId(final Long id) {
		this.id = id;
	}

	public void setName(final String name) {
		this.name = name;
	}
}

File: sample-data-repository/src/main/scala/nl/haploid/octowight/sample/data/CaptainModel.scala
package nl.haploid.octowight.sample.data

import java.lang

import nl.haploid.octowight.registry.data.Model

import scala.beans.BeanProperty

class CaptainModel extends Model {
  @BeanProperty var id: lang.Long = null
  @BeanProperty var name: String = null
}
